Conceptpower+Spring/src/main/java/edu/asu/conceptpower/rest/ConceptSearch.java

```diff
@@ -101,25 +107,20 @@ private void initBinder(WebDataBinder binder) {
 
         int page = 1;
 
+        preprocessConceptSearchParameterBean(conceptSearchParameters);
+
+        PropertyDescriptor pageDescriptor = new PropertyDescriptor("page", ConceptSearchParameters.class, "getPage",
+                "setPage");
+        if (pageDescriptor.getReadMethod().invoke(conceptSearchParameters, new Object[] {}) != null) {
+            page = Integer.parseInt(
+                    String.valueOf(pageDescriptor.getReadMethod().invoke(conceptSearchParameters, new Object[] {})));
+        }
```

Comment:
Loaded the page number from conceptsearchparamter to page variable in controller.